modular_chomp/code/modules/projectiles/ammo_refactor/ammo_types.dm

```diff
@@ -0,0 +1,132 @@
+// Projectile.
```

Comment:
Now this, this whole file, these are good. Even if you didn't simplify the entire goddamn roster of ammo for the game, the _projectiles_ are not things players will see by name.

So you could, if you wanted to, have 5.56 or 5.45 or 5mm or 5.55 or .357 or .50AE or whatever the hell have you and just arbitrarily decide "THESE ARE _ALL_ CLASSED AS DOING _MEDIUM_ BULLET DAMAGE" like this is what I tried to do in the first place years ago and you just made it so much neater and cleaner and easier to work with. I'm stealing this kind of work if I do my own gun PR. This is a much nicer way of doing it.